frontend/portals/ProductItemFavoritesButton/components/CTAButtons/components/AddToCartPicker/index.jsx

```diff
@@ -156,31 +164,59 @@ class AddToCartPicker extends Component {
   }
 
   /**
-   * Function to handle cart
+   * Function to handle products added to cart
    * @param {number} quantity quantity selected.
    */
-  handelAddToCart = (quantity) => {
-    this.props.handleAddToCart(quantity);
+  onAddToCart = async (quantity) => {
+    const {
+      isOrderable, isSimpleProduct, fetchVariants, handleAddToCart, cachedProduct,
+    } = this.props;
+    if (addFirstVariantToCart && isOrderable && !isSimpleProduct) {
+      try {
+        const cachedVariantId = cachedProduct ? cachedProduct.variants?.products[0].id : null;
+
+        const variants = await fetchVariants();
 
-    this.setState(prevState => ({
-      addedQuantity: prevState.addedQuantity + quantity,
-    }));
+        // case 1: product variant request has not been done before
+        if (variants) {
+          const variantId = variants.products[0].id;
+          handleAddToCart(quantity, variantId);
+          this.setState(prevState => ({
+            addedQuantity: prevState.addedQuantity + quantity,
+          }));
+        }
+        // case 2: product variant request has been done before and variants were cached
+        if (!variants && cachedVariantId) {
+          handleAddToCart(quantity, cachedVariantId);
+          this.setState(prevState => ({
+            addedQuantity: prevState.addedQuantity + quantity,
+          }));
+        }
+      } catch (error) {
+        console.error('Failed to fetch variants:', error);
+      }
+    } else {
+      handleAddToCart(quantity);
+      this.setState(prevState => ({
+        addedQuantity: prevState.addedQuantity + quantity,
+      }));
+    }
   };
 
   /**
    * Renders the component.
-   * @returns {JSX}
+   * @returns {JSX.Element}
    */
   render() {
     const pickerItems = createPickerItems(this.props.stock, maxEntries);
     return (
-      <BasePicker
+      <Picker
```

Comment:
The component is not accessible. The button as no label and the items of the quantity picker are set to `aria-hidden`. https://github.com/shopgate-professional-services/ext-product-list-add-to-cart/blob/master/frontend/portals/ProductItemFavoritesButton/components/CTAButtons/components/AddToCartPicker/components/List/components/Item/index.jsx#L114

i think someone just copied the `List` component from somewhere else, since it has props that are never used. since you'll now care about the List component, you can maybe fix the styling of the quantity list. it's caused by some `marginLeft: 72` / `marginLeft: -72` in the style.js of the List component.